### PR TITLE
[C-2220] Upgrade push-notification library

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Upgrades mobile push-notification library from react-native-push-notifications to react-native-notifications. Main reason for doing this is maintainability, as the `react-native-push-notifications` has an unmaintained notice and recommends `react-native-notifications` as a replacement. The other win we get with this library, is consolidating deep-links and push-notifs under one flow, which means less code and faster push-notification interactions

### Dragons

Need to make damn sure push-notifs continue to work for ios and android

### CODE REVIEW

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by nicoback2** [comment]
> It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype
1
Looks like you'd want to bump the version to 1.1.62 (unless you are cherry picking this for this week's release)!

File: packages/mobile/package.json [hunk 1]
```diff
@@ -125,9 +124,9 @@
     "react-native-markdown-display": "7.0.0-alpha.2",
     "react-native-modal": "13.0.0",
     "react-native-modal-datetime-picker": "14.0.0",
+    "react-native-notifications": "^4.3.3",
```
Comment (sliptype) [inline packages/mobile/package.json:127]: 📍

**Review by sliptype — APPROVED** [review]
Looks good to me! Yeah let's make sure to test on both platforms

And please bump the version number by following these instructions: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4#8c5d2490e68949afa5acadca8c42906e

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Pull request merged**